### PR TITLE
Fixing the AppDesigner TabButtons to show focus cues when navigation happens via Up/Down.

### PR DESCRIPTION
**Pull request by tannergooding**
**Customer scenario**

The user who completely rely on the keyboard, won't know which control has focus when using the Up/Down keys to navigate.

**Bugs this fixes:** 

https://devdiv.visualstudio.com/DevDiv/_workitems?id=409661

**Workarounds, if any**

Use Tab/Shift+Tab rather than Down/Up

**Risk**

Minimal, this just updates the code to additionally display focus cues when navigation happens via the Up/Down keys.

**Performance impact**

None

**Is this a regression from a previous update?**

No, this was missed in the original fix.

**Root cause analysis:**

Navigation via up/down was not listed as a requirement in the original bug.

**How was the bug found?**

Accessibility

### CODE REVIEW

**Comment by tannergooding** [comment]
FYI. @dpoeschl, @MattGertz for accessibility approval

**Comment by srivatsn** [comment]
Please retarget to the dev15.3.x branch, master is now a 15.5 branch

**Comment by tannergooding** [comment]
Retargeted to 15.3.x

File: src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/DesignerTabButton.vb [hunk 1]
```diff
@@ -143,20 +143,23 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
...
+
+                    If keyCode <> Keys.Tab OrElse (keyData And Keys.Control) <> Keys.Control Then                                   ' Don't process if Ctrl+Tab, but do Process for Ctrl+Up and Ctrl+Down
                         Dim parent As ProjectDesignerTabControl = ParentTabControl
                         If parent IsNot Nothing Then
                             Dim nextIndex As Int32 = ButtonIndex
-                            If (keyData And Keys.Shift) = Keys.Shift Then
+                            If (keyCode = Keys.Tab AndAlso (keyData And Keys.Shift) = Keys.Shift) OrElse keyCode = Keys.Up Then     ' Process if Shift+Tab or Up
                                 nextIndex -= 1
-                                If nextIndex <= 1 Then
+                                If nextIndex < 0 Then
```
Comment (srivatsn) [inline src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/DesignerTabButton.vb]: Is this right? why this change?

Comment (tannergooding) [inline src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/DesignerTabButton.vb]: It should be `< 0`, otherwise it cycles from the second tab item to the last and will never hit index 0 or 1 (when cycling in reverse).
